### ISSUE TITLE
BUG: Focus wrong on load

### ISSUE THREAD

**Issue by larsoner** [mne-tools/mne-qt-browser#10]
```
MNE_BROWSE_BACKEND=pyqtgraph mne browse_raw ~/mne_data/MNE-sample- 
data/MEG/sample/sample_audvis_raw.fif
```
I get:
![Screenshot from 2021-11-01 10-55-47](https://user-images.githubusercontent.com/2365790/139691961-2da82469-58e2-4bfa-82d9-d47a4e70b6b7.png)

Pressing rightarrow is correct (scrolls right), but pressing downarrow is incorrect as it changes the overview bar mode rather than scrolling down the channels:

![Screenshot from 2021-11-01 10-57-12](https://user-images.githubusercontent.com/2365790/139692206-5814a581-5bfa-4199-99c6-92df5dc60948.png)

In other words, I think the overview bar has keyboard focus on init when it should be the data view.

**Comment by marsipu** [comment]
Thank you @larsoner, the ComboBox captured the keyboard-focus. I fixed that now.
